src/App.vue

```diff
@@ -0,0 +1,253 @@
+<script setup>
+import { ref, onMounted } from 'vue'
+
+const origins = ref([])
```

Comment:
specify type for origins, the inferred type will be `never[]`

```suggestion
const origins = ref<string[]>([])
```

Reply:
Resolved, will raise a commit on this PR soon.